src/class/Transpiler.ts

```diff
@@ -1236,6 +1289,12 @@ export class Transpiler {
 				}
 				initializers.forEach(initializer => (result += this.indent + initializer + "\n"));
 				result += this.transpileBlock(body);
+
+				for (const child of node.getStatements()) {
+					if (ts.TypeGuards.isReturnStatement(child)) {
+						throw new TranspilerError(`Cannot use return statement in constructor for ${className}`, child)
+					}
+				}
```

Comment:
I'd rather change this to something like:
```TS
const retrunStatement = node.getStatementByKind(ts.SyntaxKind.ReturnStatement);
if (retrunStatement) {
	throw new TranspilerError(`Cannot use return statement in constructor for ${className}`, retrunStatement);
}
```

Reply:
Done!